src/VisualStudio/Core/Test.Next/project.json

```diff
@@ -2,18 +2,8 @@
   "dependencies": {
     "Newtonsoft.Json": "8.0.3",
     "Moq": "4.2.1402.2112",
```

Comment:
are you on slack heejae?  i want to add you to a swat discussion i've started on this.